src/SQLChecks/Functions/Public/Get-AutoStartupSQLServices.ps1

```diff
@@ -23,5 +23,10 @@ where
 and startup_type_desc <> 'Automatic';
 "@
 
-    (Invoke-SQLCMD -TrustServerCertificate -ServerInstance $serverInstance -Query $query -Database master).servicename
+    $services =@(Invoke-SQLCMD -TrustServerCertificate -ServerInstance $serverInstance -Query $query -Database master) |
```

Comment:
Can y'all repro a situation where the left code returns a different output than the right?